packages/sources/src/components/card/card.stories-of.ts

```diff
@@ -27,7 +27,8 @@ export function storiesOfCard<TemplateFnReturnType>(
               icon: 'info'
             }
           }
-        ]
+        ],
+        imgSrc: ''
```

Comment:
De opzet van deze Story is niet heel duidelijk, maar wat je hiermee doet is een default value van een lege string zetten. Dat is iets anders dan `undefined`. Ik denk dat je dit op `undefined` moet zetten, of op regel 18 iets met `Omit` moet doen.

Reply:
Ik heb gekozen voor de `Omit` oplossing